inference-engine/src/vpu/graph_transformer/src/stages/proposal.cpp

```diff
@@ -128,8 +131,15 @@ void FrontEnd::parseProposal(const Model& model, const ie::CNNLayerPtr& layer, c
     // #-37327: Several models Failed to compile layer "proposals"
     IE_ASSERT(outputs.size() == 1 || outputs.size() == 2);
```

Comment:
If it would not be hard for you, please, replace `IE_ASSERT` with preferable in vpu plugin `VPU_THROW_UNLESS`

Reply:
done

Reply:
@avershov, I can't see any changes :( Are you sure that you've updated the commits?

Reply:
committed; and I hope yet another firmware update commit